### DISCUSSION TITLE
Account lost following the lost of smartphone and computer

### DISCUSSION THREAD

**Discussion by zerocoolmong** [community/community#138675] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,

I recently realized that I lost access to my main account @MongZerocool because I used "Authenticator" app in a phone I recently changed. Unfortunately I alsolost my laptop at the same time and don't have recovery codes anymore.
The only thing I can confirm is my email address or my phone number.

It looks like my GitHub account is definitively lost, am I right ?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @zerocoolmong, 

Thank you for your patience. The community is unable to assist with nor escalate with this account-related situation. Please review our doc: [Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). Within the doc, you also have the option to go through recovery with a Virtual Agent. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by queenofcorgis** [reply]
@abharga2 - this option may help you based on what we discussed!
